README.md

```diff
@@ -1,63 +1,82 @@
+  - Pictures can be directly downloaded from the gallery.
   - Print feature.
-    - Optional: Print a frame on your picture (replace resources/img/frames/frame.png with a proper frame).
+    - Optional: Print a frame on your picture
+      (replace `resources/img/frames/frame.png` with a frame of your choice).
     - Optional: Print text on your picture.
     - Optional: Print QR-Code on the right side of your picture.
-  - Pictures can be send via E-Mail.
+  - Pictures can be sent via e-mail.
   - LivePreview (uses device cam).
-  - Event (e.g. wedding, birthday) specifig config to show a symbol (e.g. heart) betweeen some text on the startpage.
+  - Event specific (e.g. wedding, birthday) config to show a symbol (e.g. heart)
+    between some text on the start page.
   - Green screen keying (chroma keying).
-  - Photo collage function: take 4 pictures in a row with or without interruption and let it generate a collage out of it.
-  - Save pictures with a polaroid effect.
+  - Photo collage function: take 4 pictures in a row with or without
+    interruption and let it generate a collage out of it.
+  - Save pictures with a Polaroid effect.
   - Adjust take picture and print commands.
   - Optional Blue-gray theme.
   - And many more options to adjust Photobooth for your personal needs.
   
 ## :camera: Screenshots
+
 ![](https://raw.githubusercontent.com/wiki/andreknieriem/photobooth/images/start.png)
 
 ## :gear: Prerequisites
+
 - gphoto2 installed, if used on a Raspberry for DSLR control
 - digiCamControl, if used unter Windows for DSLR control
 - NGINX, Lighttpd or Apache
 
 ## :wrench: Installation & Troubleshooting
-Please follow the installation instructions in our [Photobooth-Wiki](https://github.com/andreknieriem/photobooth/wiki) to setup Photobooth.
 
-If you're having trouble or questions please take a look at our [FAQ](https://github.com/andreknieriem/photobooth/wiki#faq---frequently-asked-questions) before opening a new issue.
+Please follow the installation instructions in our
+[Photobooth-Wiki](https://github.com/andreknieriem/photobooth/wiki) to setup
+Photobooth.
+
+If you're having trouble or questions please take a look at our
+[FAQ](https://github.com/andreknieriem/photobooth/wiki#faq---frequently-asked-questions)
+before opening a new issue.
 
 ### :mag: Changelog
+
 Please take a look at the changelog in our [Photobooth Wiki](https://github.com/andreknieriem/photobooth/wiki/changelog).
 
 ### :mortar_board: Tutorial
+
 [Raspberry Pi Weddingphotobooth (german)](https://www.andrerinas.de/tutorials/raspberry-pi-einen-dslr-weddingphotobooth-erstellen.html)
 
 ### :clap: Contributors and thanks to
+
 - [dimsemenov](https://github.com/dimsemenov/photoswipe) for photoswipe
 - [t0k4rt](https://github.com/t0k4rt/phpqrcode) for phpqrcode
-- [nihilor](https://github.com/nihilor/photobooth) for Printing feature, code rework and bugfixes
+- [nihilor](https://github.com/nihilor/photobooth) for Printing feature,
```

Comment:
Do we really have to name the contributions? If someone likes to see who did what he can look at https://github.com/andreknieriem/photobooth/graphs/contributors

Reply:
It was there already, but sure we can remove that too